lightstep/tracer.py

```diff
@@ -75,6 +76,7 @@ def __init__(self, enable_binary_format, recorder, scope_manager):
         super(_LightstepTracer, self).__init__(recorder, scope_manager=scope_manager)
         self.register_propagator(Format.TEXT_MAP, TextPropagator())
         self.register_propagator(Format.HTTP_HEADERS, TextPropagator())
+        self.register_propagator(B3, B3Propagator())
```

Comment:
Let's leave this as an optional step (so by default only `TextMapPropagator` is registered by default).

Reply:
Done, removed `B3` as a default propagator.